backend/apps/blogs/resolvers.py

```diff
@@ -0,0 +1,12 @@
+from .models import BlogPost as BlogPostModel
+
+
+class BlogPostResolvers:
+    def resolve_all_blog_posts(self, info):
+        return BlogPostModel.objects.all()
+
+    def resolve_blog_post_by_id(self, info, blog_post_id):
+        try:
+            return BlogPostModel.objects.get(pk=blog_post_id)
+        except BlogPostModel.DoesNotExist:
+            return None
```

Comment:
I know this is also how it's done elsewhere, but I suggest going this route instead:
```suggestion
from typing import Optional

from apps.blogs import models


class BlogPostResolvers:
    def resolve_all_blog_posts(self, info):
        return models.BlogPost.objects.all()

    def resolve_blog_post_by_id(self, info, blog_post_id: int) -> Optional[models.BlogPost]:
        try:
            return models.BlogPost.objects.get(pk=blog_post_id)
        except models.BlogPost.DoesNotExist:
            return None
```